Disable libnss-db as the format is not compatible with the libnss-db package, and is
architecture dependent.

---
 nss/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/nss/Makefile
+++ b/nss/Makefile
@@ -61,7 +61,7 @@
 endif
 
 # Specify rules for the nss_* modules.  We have some services.
-services		:= files db compat
+services		:= files compat
 
 extra-libs		= $(services:%=libnss_%)
 # These libraries will be built in the `others' pass rather than
